h2o-r/h2o-package/R/kvstore.R

```diff
@@ -145,6 +145,8 @@ h2o.getModel <- function(model_id) {
   MetricsClass <- paste0("H2O", model_category, "Metrics")
   # setup the metrics objects inside of model...
   model$training_metrics   <- new(MetricsClass, algorithm=json$algo, on_train=TRUE, on_valid=FALSE, on_xval=FALSE, metrics=model$training_metrics)
+  model$var.names$domains <- json$output$domains
```

Comment:
Names and domains are excluded on like 144, is there a good reason for it?

Is the best thing to do just to remove them from the exclude on line 144?

Reply:
It is. Removed, excluded.